src/App.js

```diff
@@ -0,0 +1,50 @@
+import "./App.css";
+
+import { Route, Routes } from "react-router-dom";
+import MainLayout from "./layouts/MainLayout";
+import Cards from "./components/Cards/Cards";
+import Caseload from "./pages/Caseload/Caseload";
+function App() {
+  return (
+    <div className="App">
+      <div className="app-glass">
+        <Routes>
+          <Route
+            path="/caseload"
+            element={
+              <MainLayout>
+                <Caseload />
+              </MainLayout>
+            }
+          />
```

Comment:
It's better to use `MainLayout` inside the `CaseLoad` component. So here it will be 
```suggestion
             element={<Caseload />}
```